src/components/file/file.tsx

```diff
@@ -131,38 +160,28 @@ export class File {
     }
 
     public render() {
-        const chipArray = this.value
-            ? [
-                  {
-                      ...DEFAULT_FILE_CHIP,
-                      text: this.value.filename,
-                      id: this.value.id,
-                  },
-              ]
-            : [];
-
         return [
             <input
+                hidden={true}
                 id={this.fileInputId}
-                type="file"
                 onChange={this.handleFileChange}
-                hidden={true}
+                type="file"
             />,
             <limel-chip-set
                 disabled={this.disabled}
                 label={this.label}
-                required={this.required}
-                onKeyDown={this.handleKeyDown}
-                onKeyUp={this.handleKeyUp}
-                type="input"
-                value={chipArray}
-                onClick={this.handleFileSelection}
+                leadingIcon="upload_to_cloud"
                 onChange={this.handleChipSetChange}
-                onInteract={this.preventAndStop}
-                onDrop={this.handleFileDrop}
+                onClick={this.handleFileSelection}
                 onDragEnter={this.preventAndStop}
                 onDragOver={this.preventAndStop}
-                leadingIcon="upload_to_cloud"
+                onDrop={this.handleFileDrop}
+                onInteract={this.preventAndStop}
+                onKeyDown={this.handleKeyDown}
+                onKeyUp={this.handleKeyUp}
+                required={this.required}
+                type="input"
+                value={this.chipArray}
```

Comment:
Are you using some automatic formatter that sorts attributes alphabetically?

I don't mind having them sorted alphabetically, but then we need to have that done by prettier, so we don't get commits like this, with one small change, and lots of completely unrelated stylistic changes… 😕

You'll have to either remove these changes, or put them in their own commit (it can still be in this PR, but must be a separate commit).

Reply:
Sorry, my ocd just can't stand unsorted properties (and imports) so I do that manually but agree it should be done by the formatter. I will fix it.

Reply:
Should be good now @adrianschmidt

Reply:
For the javascript imports, the order probably doesn't matter, but please note that in the scss files, the import order is important, so please don't change the order of imports there…